### PR TITLE
finished user story 18 closes #8

### LINKED ISSUE #8: User Story 18, User can delete a book review
```
As a Visitor,
When I visit a user's show page,
I see a link next to each review to delete the review.
When I delete a review I am returned to the user's show page
Then I should no longer see that review.
```

### Checklist:

- [ ] base code tests are written
- [ ] base code is written to pass tests
- [ ] edge case ("sad path") tests written
- [ ] edge case code written
- [ ] code is reviewed
- [ ] branch is merged
- [ ] all tests still pass

child of #9

child of #33

depends on #16

depends on #11

### PR DESCRIPTION
**Pull request by earl-stephens**
Completed user story 18.  Have ability to delete a review.  If that review is the only one for that user, then the app redirects back to the book index page, since there will no longer be a user show page.  Tests pass.  Closes #8

### CODE REVIEW

File: app/views/users/show.html.erb [hunk 1]
```diff
@@ -6,4 +6,5 @@
     <p id="book_title_path<%= review_info.id %>">Book Title: <%= link_to "#{review_info.book.title}", book_path(review_info.book.id) %></p>
     <img src="<%= review_info.book.image %>" alt="book image"/>
     <p>Date of Review: <%= review_info.created_at %></p>
+    <%= link_to "Delete Review: #{review_info.title}", review_path(review_info.id), method: :delete, data: { confirm: "Are you sure?"} %>
```
Comment (djc00p) [inline app/views/users/show.html.erb]: would like to go over this bit of code

**Comment by djc00p** [comment]
looks good
